cmd/minishift/cmd/start.go

```diff
@@ -419,6 +425,11 @@ func initClusterUpFlags() *flag.FlagSet {
 	clusterUpFlagSet.AddFlag(cmdutil.HttpProxyFlag)
 	clusterUpFlagSet.AddFlag(cmdutil.HttpsProxyFlag)
 
+	if future {
```

Comment:
I don't mean hiding in general. If you enable experimental features, we of course show them. However, by adding them in hidden, I believe we should also be able to support an actual '--enable-experimental-flags' option.